lib/resources/majors.tsx

```diff
@@ -2711,6 +2711,308 @@ const bsMolCell: Major = {
   ],
 };
 
+const bsNeuro: Major = {
+  degree_name: 'B.S. Neuroscience',
+  abbrev: 'B.S. Neuro',
+  department: 'AS Neuroscience',
+  total_degree_credit: 120,
+  wi_credit: 12,
+  url: 'https://e-catalogue.jhu.edu/arts-sciences/full-time-residential-programs/degree-programs/neuroscience/neuroscience-bachelor-science/',
+  distributions: [
+    {
+      name: 'Neuroscience Sequence',
+      required_credits: 12,
+      min_credits_per_course: 3,
+      description:
+        'These are core neuroscience courses required for the major. ' +
+        'For more information please visit the ' +
+        "<a href='https://krieger.jhu.edu/neuroscience/bs-program/requirements/'>" +
+        'major degree requirement</a> section on the department website.',
+      criteria:
+        'AS.200.141[C]^OR^AS.050.105[C]^OR^AS.050.203[C]^OR^AS.080.250[C]^OR^AS.080.305[C]^OR^AS.080.306[C]',
+      fine_requirements: [
+        {
+          description:
+            '<b>Introductory Neuroscience</b> <br /> AS.200.141 Foundations of Brain, Behavior and Cognition' +
+            ' <br /><i>OR</i> <br />AS.050.105 Introduction to Cognitive Neuropsychology',
+          required_credits: 0,
+          criteria: 'AS.200.141[C]^OR^AS.050.105[C]',
+        },
```

Comment:
Remove Prerequisite classes from fine requirements of the Neuroscience Sequence Distribution.